app/controllers/group_sets_controller.rb

```diff
@@ -11,7 +11,11 @@ def load_resource
       when :index
         @group_sets = GroupSet.all.desc(:created_at).page(params[:page]).per(10)
       when :show
-        @group_set = GroupSet.find(params[:id])
+        if request.fullpath == "/"
```

Comment:
こちら以下のようにかけると思うので試してみてください！
```suggestion
        if request.fullpath == root_path
```

Reply:
@narukami894 対応いたしました！